cli/internal/runcache/runcache.go

```diff
@@ -225,8 +227,10 @@ func (tc TaskCache) OutputWriter() (io.WriteCloser, error) {
 	if err != nil {
 		return nil, err
 	}
+	pref := tc.colorCache.PrefixColor(tc.pt.PackageName)
+	actualPrefix := pref(tc.pt.OutputPrefix())
 	bufWriter := bufio.NewWriter(output)
-	if _, err := bufWriter.WriteString(fmt.Sprintf("%s: cache hit, replaying output %s\n", tc.pt.OutputPrefix(), ui.Dim(tc.hash))); err != nil {
+	if _, err := bufWriter.WriteString(fmt.Sprintf("%s: cache hit, replaying output %s\n", actualPrefix, ui.Dim(tc.hash))); err != nil {
```

Comment:
Hmm, are we already baking the color into the log files?

Reply:
We are not: https://github.com/vercel/turborepo/blob/1cfaff0f5152cea02e98f872e82d5e72a167d1f4/cli/internal/run/run.go#L827